src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -51,6 +53,23 @@ const MetaTraderConfig = (() => {
                     }
                 })
             ),
+            pre_submit: () => (
+                new Promise((resolve) => {
+                    if (State.getResponse('landing_company.gaming_company.shortcode') === 'malta') {
+                        Dialog.confirm({
+                            id     : 'confirm_new_account',
+                            message: ['Trading Contracts for Difference (CFDs) on Volatility Indices may not be suitable for everyone. Please ensure that you fully understand the risks involved, including the possibility of losing all the funds in your MT5 account. Gambling can be addictive – please play responsibly.', 'Do you wish to continue?'],
```

Comment:
do we need to localize dialog messages?

Reply:
@ruskakimov it's done here internally https://github.com/binary-com/binary-static/pull/3651/files#diff-5dc08385bbeed4b2449ee289beabac9eR22